vsts_cd_manager/continuous_delivery_manager.py

```diff
@@ -225,14 +223,14 @@ def _get_build_configuration(self, app_type_details):
 
     def _get_source_repository(self, uri, token, branch, cred, username, password):
         # Determine the type of repository (TfsGit, github, tfvc, externalGit)
-        # Find the identifier and set the properties; default to externalGit
+        # Find the identifier and set the properties.
+        # Default is externalGit
         type = 'Git'
         identifier = uri
         account_name = None
         team_project_name = None
-        auth_info = None
-        if username is not None and password is not None: 
-            auth_info = AuthorizationInfo('UsernamePassword', AuthorizationInfoParameters(None, None, username, password))
+        auth_info = AuthorizationInfo('UsernamePassword', AuthorizationInfoParameters(None, None, username, password))
```

Comment:
Why this change?

Reply:
This is default auth_info for the case of External Git. No matter if username/password has some value or not, authorization_scheme will always be Username/Password. Verified the scenario with Ibiza as well.
Also, this particular change is already reviewed and pushed.
https://github.com/Microsoft/vsts-cd-manager/pull/18